nupic/utils.py

```diff
@@ -85,16 +85,15 @@ def getSlidingWindow(self):
     return self.slidingWindow
 
 
-  def __setstate__(self, state):
```

Comment:
Yes, @breznak, we must have a period of time when both serialization methods are working so people can migrate.